src/app/tests/suites/certification/Test_TC_LVL_2_1.yaml

```diff
@@ -62,6 +62,8 @@ tests:
           saveAs: MinLevelValue
           constraints:
               type: int8u
+              minValue: 1
```

Comment:
Actually, for all the lights (i.e. all the device types with LT in Matter spec), the minvalue has to be 1, and the maxValue has to be 254. This check should be done by a Python test.